src/nvme/types.h

```diff
@@ -1854,27 +1854,34 @@ enum nvme_id_nsfeat {
  * enum nvme_id_ns_flbas - This field indicates the LBA data size & metadata
  *			   size combination that the namespace has been
  *			   formatted with
- * @NVME_NS_FLBAS_LOWER_MASK:	Mask to get the index of one of the supported
- *				LBA Formats's least significant
- *				4bits indicated in
- *				:c:type:`struct nvme_id_ns <nvme_id_ns>`.lbaf.
- * @NVME_NS_FLBAS_META_EXT:	Applicable only if format contains metadata. If
- *				this bit is set, indicates that the metadata is
- *				transferred at the end of the data LBA, creating an
- *				extended data LBA. If cleared, indicates that all
- *				of the metadata for a command is transferred as a
- *				separate contiguous buffer of data.
- * @NVME_NS_FLBAS_HIGHER_MASK:	Mask to get the index of one of
- *				the supported LBA Formats's most significant
- *				2bits indicated in
- *				:c:type:`struct nvme_id_ns <nvme_id_ns>`.lbaf.
+ * @NVME_NS_FLBAS_LOWER_MASK:	 Mask to get the index of one of the supported
+ *				 LBA Formats's least significant
+ *				 4bits indicated in
+ *				 :c:type:`struct nvme_id_ns <nvme_id_ns>`.lbaf.
+ * @NVME_NS_FLBAS_META_EXT_MASK: Applicable only if format contains metadata. If
+ *				 this bit is set, indicates that the metadata is
+ *				 transferred at the end of the data LBA, creating an
+ *				 extended data LBA. If cleared, indicates that all
+ *				 of the metadata for a command is transferred as a
+ *				 separate contiguous buffer of data.
+ * @NVME_NS_FLBAS_HIGHER_MASK:	 Mask to get the index of one of
+ *				 the supported LBA Formats's most significant
+ *				 2bits indicated in
+ *				 :c:type:`struct nvme_id_ns <nvme_id_ns>`.lbaf.
  */
 enum nvme_id_ns_flbas {
-	NVME_NS_FLBAS_LOWER_MASK	= 15 << 0,
-	NVME_NS_FLBAS_META_EXT		= 1 << 4,
-	NVME_NS_FLBAS_HIGHER_MASK	= 3 << 5,
+	NVME_NS_FLBAS_LOWER_SHIFT	= 0,
+	NVME_NS_FLBAS_META_EXT_SHIFT	= 4,
+	NVME_NS_FLBAS_HIGHER_SHIFT	= 5,
+	NVME_NS_FLBAS_LOWER_MASK	= 0xf,
+	NVME_NS_FLBAS_META_EXT_MASK	= 0x1,
+	NVME_NS_FLBAS_HIGHER_MASK	= 0x3,
 };
 
+#define NVME_NS_FLBAS_LOWER(flbas)	NVME_GET(flbas, NS_FLBAS_LOWER)
+#define NVME_NS_FLBAS_META_EXT(flbas)	NVME_GET(flbas, NS_FLBAS_META_EXT)
```

Comment:
The change of `NVME_NS_FLBAS_META_EXT` from a simple define to a macro will break existing users. While for nvme-cli that might be okay as we have to update the minimum dependency to libnvme anyway, it's not so easy to tell about other users. I am aware of that the udisk and nvme-stats project is using libnvme but not this define. 

So what we can do is merge it but if someone complains we are reverting it. What do you think?